Details on CSV locations and constraints (#744)
cap-js/docs: guides/databases-hana.md

### CSV Data Gets Overridden

> HDI deploys CSV data as _.hdbtabledata_ located in folder _db/src/**_. HDI assumes exclusive ownership of the data. It's overridden with the next application deployment; hence:

Reviewer: Mentioning `db/src` location is misleading a bit, as it also true for `gen/db/...`
I would just not mention it here.  The statement here is that any `hdbtabledata` file has this behavior, regardless from where it was deployed (generated, manual, etc).

> HDI deploys CSV data as _.hdbtabledata_ located in folder _db/src/**_. HDI assumes exclusive ownership of the data. It's overridden with the next application deployment; hence:

Reviewer: ```suggestion
HDI deploys CSV data as _.hdbtabledata_ and assumes exclusive ownership of the data. It's overridden with the next application deployment; hence:
```
